cmd/cosign/cli/verify/verify.go

```diff
@@ -161,6 +163,21 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 			return fmt.Errorf("getting Fulcio intermediates: %w", err)
 		}
 	}
+	if !co.SkipTlogVerify {
```

Comment:
Rekor client init is done above. We just need to remove the duplication.

Reply:
Ah thanks!

Reply:
done!